srcpkgs/gnome-weather/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'gnome-weather'
 pkgname=gnome-weather
-version=44.0
+version=45.0
 revision=1
 build_helper="gir"
 build_style=meson
-hostmakedepends="gettext pkg-config itstool glib-devel gjs"
+hostmakedepends="gettext pkg-config itstool glib-devel gjs desktop-file-utils
+ gtk-update-icon-cache"
```

Comment:
Use gtk4-update-icon-cache, since this application already depends on gtk4.

Reply:
fixed